src/reset/HtmlButton/HtmlButton.tsx

```diff
@@ -27,7 +27,7 @@ const buttonResets = css`
   ${resets.common}
   display: inline-block;
   max-width: 100%;
-  pointer: cursor;
```

Comment:
stylelint (or similar) would prevent these kind of mistakes

Reply:
Editor informed the error but need to configure for webpack also